data/hai/hai reveal 4 middle.txt

```diff
@@ -921,7 +924,7 @@ mission "Hai Reveal [A07] Deliver the Relays"
 			`	Terry looks between you, "Glad you know who this is David. The Captain here is doing a lot to help us out, so please, make yourself acquainted. Don't keep them too long though, they've a job to do when they leave here." With that, she gives the both of you a parting nod and strides back into the chaos. Almost immediately a swarm of people with datapads seem to materialize from nowhere and pursue her.`
 			`	The man grins and shakes his head as she departs. "Remarkable woman. Anyway, the name's David Joseph Turner, venture capitalist, entrepreneur, and famous Paradise Worlds businessman." Turner grabs a business card from his pocket and hands it to you. The card contains his contact information, an address on Martini, and makes sure to boast in golden letters his net worth of more than three billion credits.`
 			`	"You may not have heard of me, but I've sure as hell heard of you." Turner turns around and waves his arm across the crowd of people working away on the station concourse. "To be involved in funding and producing this movie though, what an adventure. You know I've paid for movies to be made, or at least partially invested in them, but I've never had to actually be there while it was happening. Quite remarkable."`
-			`<Turner takes a moment to enthuse about the movie and relay a brief synopsis to demonstrate how well the production is in fact going.>`
+			`	Turner takes a moment to enthuse about the movie and relay a brief synopsis to demonstrate how well the production is in fact going.`
```

Comment:
Also a to-do list item to actually insert the conversation, but I suppose this one is fair enough.

Reply:
I'll add a "TODO" comment above it to make that clear.